src/plugins/lvm-dbus.c

```diff
@@ -2290,6 +2306,7 @@ BDLVMLVdata** bd_lvm_lvs (const gchar *vg_name, GError **error) {
     lv = matched_lvs;
     while (lv) {
         props = get_object_properties (lv->data, LV_CMN_INTF, error);
+        g_free(lv->data);
```

Comment:
Hmm, there's a risk of leaking memory after first iteration since the rest of the items wouldn't be freed. Better to replace all `g_slist_free()` calls with `g_slist_free_full()`.

Plus missing space before opening parenthesis :-)

Reply:
Nice catch. I've updated the PR to use ``g_slist_free_full(matched_lvs, g_free)``